providers/edge3/src/airflow/providers/edge3/cli/edge_command.py

```diff
@@ -787,6 +803,13 @@ def remote_worker_request_shutdown(args) -> None:
     help="State of the edge worker",
 )
 
+ARG_DAEMON = Arg(
+    ("-D", "--daemon"), help="Daemonize instead of running in the foreground", action="store_true"
+)
+ARG_STDERR = Arg(("--stderr",), help="Redirect stderr to this file")
+ARG_STDOUT = Arg(("--stdout",), help="Redirect stdout to this file")
+ARG_LOG_FILE = Arg(("-l", "--log-file"), help="Location of the log file")
```

Comment:
What is the difference of `-l` to `--stdout/err`? Is this just a convenience that you don't need to set both?
If these are not given, will stdout/err just be directed to /dev/null?
Should these options carry a default to make logs into the working directory where started?

Reply:
the `run_command_with_daemon_option` works as such. When -D is unspecified the -l/stderr/out are ignored. When running as a daemonized process. If the log files are unspecified, it will create `airflow-edge-worker.pid,log,out,err` files in $AIRFLOW_HOME. The difference between -l and --stdout is that -l only captures output from the logger object, however --stdout will capture entire stdout ( including prints)

Reply:
As the parameters are only used when in daemon mode, can you mention this in the help?
```suggestion
ARG_STDERR = Arg(("--stderr",), help="Redirect stderr to this file if run in daemon mode")
ARG_STDOUT = Arg(("--stdout",), help="Redirect stdout to this file if run in daemon mode")
ARG_LOG_FILE = Arg(("-l", "--log-file"), help="Location of the log file if run in daemon mode")
```